jupyterhub_moss/spawner.py

```diff
@@ -131,6 +133,11 @@ def _slurm_info_resources(self, slurm_info_out):
                 gpus_total = 0
                 gpu = None
 
+            max_runtime = parse_timelimit(timelimit)
+            if max_runtime is None:
+                # Parsing failed: Limit to one day
```

Comment:
Would perhaps be nice to log this as a warning

Reply:
Yes, definitely